base/src/main/java/com/smartdevicelink/proxy/rpc/TireStatus.java

```diff
@@ -179,56 +225,128 @@ public WarningLightStatus getPressureTelltale() {
         return (WarningLightStatus) getObject(WarningLightStatus.class, KEY_PRESSURE_TELL_TALE);
     }
 
-    public TireStatus setLeftFront(@NonNull SingleTireStatus leftFront) {
+    /**
+     * Sets the leftFront.
+     *
+     * @param leftFront The status of the left front tire.
+     * @since SmartDeviceLink 2.0.0
+     */
+    public TireStatus setLeftFront(SingleTireStatus leftFront) {
         setValue(KEY_LEFT_FRONT, leftFront);
         return this;
     }
 
+    /**
+     * Gets the leftFront.
+     *
+     * @return SingleTireStatus The status of the left front tire.
+     * @since SmartDeviceLink 2.0.0
+     */
     public SingleTireStatus getLeftFront() {

```

Comment:
So getters like these now could possibly return null objects right? Is there a place where this can be notified to the developer to be cautious?

Reply:
Added the @Nullable modifier.

Reply:
Added text comment: 'Optional since SmartDeviceLink 8.0.0'